buildkite/src/Jobs/Test/RosettaIntegrationTests.dhall

```diff
@@ -13,9 +13,14 @@ let Size = ../../Command/Size.dhall
 let Libp2p = ../../Command/Libp2pHelperBuild.dhall
 let DockerImage = ../../Command/DockerImage.dhall
 let DebianVersions = ../../Constants/DebianVersions.dhall
+
 let dirtyWhen = [ 
-  S.strictlyStart (S.contains "src/app/rosetta")
-  , S.strictlyStart (S.contains "buildkite/src/Jobs/Test/RosettaIntegrationTests")
+  S.strictlyStart (S.contains "src/app/rosetta"),
+  S.strictlyStart (S.contains "src/lib"),
+  S.strictlyStart (S.contains "src/app/archive"),
+  S.strictlyStart (S.contains "src/app/cli"),
```

Comment:
Could you confirm if this is needed and if not, remove?

Reply:
These should be needed to catch changes in the archive / node that could break Rosetta implementation.

Reply:
@deepthiskumar yes i might carry away with my `dirtyWhen` expansion. `src/app/cli` looks like it is not needed